src/Inventory.zig

```diff
@@ -1996,15 +2008,7 @@ pub fn getAmount(self: Inventory, slot: usize) u16 {
 }
 
 pub fn save(self: Inventory, allocator: NeverFailingAllocator) ZonElement {
```

Comment:
Again, in my opinion the base64 stuff should be handled externally, since it's only supposed to be used in one place, for player data.